src/api.ts

```diff
@@ -292,18 +304,70 @@ function generateOptionsString(input?: GetINCHIOptions | GetINCHIExOptions): str
 
 // #region Public Functions
 
+/**
+ * Check if the string represents valid InChIKey
+ * @param input A source InChIKey string
+ * @returns One amongst -1 , 0 , 1 , 2 , 3
+ *  - -1: InChIKey is valid and non-standard
+ *  - 0: InChIKey is valid and standard
+ *  - 1: InChIKey has invalid length
+ *  - 2: InChIKey has invalid layout
+ *  - 3: InChIKey has invalid version number (not equal to 1)
+ */
 export function CheckINCHIKey(input: string): CheckINCHIKeyReturnCode {
   return INCHIAPI.CheckINCHIKey(input) as CheckINCHIKeyReturnCode;
 }
 
+/**
+ * CheckINCHI
+ * Description: Check if the string represents valid InChI/standard InChI
+ * Input: takes two argument
+ *  input - source InChI string ,
+ *  strict (optional) - if false, just briefly check for proper layout (prefix, version, etc.)
+ * Output: Returns code -1 , 0 , 1 , 2 , 3 , 4
+ * -1: InChI is valid and non-standard
+ *  0: InChI is valid and standard
+ *  1: InChI has invalid prefix
+ *  2: InChI has invalid version number (not equal to 1)
+ *  3: InChI has invalid layout
+ *  4: Checking InChI through InChI2InChI either failed or produced a result which does not match the source InChI string
```

Comment:
Study my last commit and update accordingly. 

Upon hovering over the function name, nicely formatted text should appear.

Example:

![Screenshot 2021-08-31 at 4 02 35 PM](https://user-images.githubusercontent.com/25290212/131487693-e78630db-e2e9-4e99-9a95-a15cc93ae247.png)